best_practices.rst

```diff
@@ -238,7 +238,7 @@ Use Dependency Injection to Get Services
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you extend the base ``AbstractController``, you can only get access to the most
-common services (e.g ``twig``, ``router``, ``doctrine``, etc.), directly from the
+common services (e.g. ``twig``, ``router``, ``doctrine``), directly from the
```

Comment:
I think this should become

```suggestion
common services (e.g. ``twig``, ``router`` and ``doctrine``), directly from the
```

same for all other changes where you removed "etc" on lists following "e.g."

Reply:
Even with two examples? 

```
A:  (e.g. twig, router)
B:  (e.g. twig and router)
C:  (e.g. twig or router)
```

Reply:
Yes, I think so.

Reply:
I'll do it right now then